Text/HealthcareTA/azuredeploy.json

```diff
@@ -0,0 +1,143 @@
+      "location": "[resourceGroup().location]",
+      "sku": {
+        "name": "S"
+      },
+      "kind": "TextAnalytics",
+      "properties": {
+        "customSubDomainName": "[variables('textAnalyticsName')]",
+        "publicNetworkAccess": "Enabled"
+      }
+    },
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "2017-10-01",
+      "name": "[variables('storageAccount')]",
+      "location": "[resourceGroup().location]",
+      "sku": {
+        "name": "[parameters('storageAccountType')]"
+      },
+      "kind": "Storage"
+    },
+    {
+      "type": "Microsoft.Web/sites",
+      "apiVersion": "2016-08-01",
+      "name": "[variables('functionAppName')]",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount'))]",
+        "[resourceId('Microsoft.CognitiveServices/accounts', variables('textAnalyticsName'))]"
+      ],
+      "kind": "functionapp",
+      "properties": {
+        "name": "[variables('functionAppName')]",
+        "kind": "functionapp",
+        "httpsOnly": true,
+        "siteConfig": {
+          "appSettings": [
+            {
+              "name": "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING",
+              "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount')), '2015-06-15').key1)]"
+            },
+            {
+              "name": "WEBSITE_CONTENTSHARE",
+              "value": "[toLower(variables('functionAppName'))]"
+            },
+            {
+              "name": "AzureWebJobsDashboard",
+              "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount')), '2015-06-15').key1)]"
+            },
+            {
+              "name": "AzureWebJobsStorage",
+              "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount')), '2015-06-15').key1)]"
+            },
+            {
+              "name": "FUNCTIONS_EXTENSION_VERSION",
+              "value": "~2"
+            },
+            {
+              "name": "FUNCTIONS_EXTENSION_RUNTIME",
+              "value": "dotnet"
+            },
+            {
+              "name": "SCM_DO_BUILD_DURING_DEPLOYMENT",
+              "value": true
+            },
+            {
+              "name": "PROJECT",
+              "value": "[variables('functionProject')]"
+            },
+            {
+              "name": "HEALTHCARE_API_KEY",
+              "value": "TODO"
+            },
+            {
+              "name": "HEALTHCARE_API_ENDPOINT",
+              "value": "TODO"
+            }
```

Comment:
At least for the key, you should be able to do this using the listkeys function. There is an example of this in the JFK sample:

[https://github.com/microsoft/AzureSearch_JFK_Files/blob/9ba42347a6285ce0b73b5bcb935e8c84852f9cfc/azuredeploy.json#L250](https://github.com/microsoft/AzureSearch_JFK_Files/blob/9ba42347a6285ce0b73b5bcb935e8c84852f9cfc/azuredeploy.json#L250)

Another useful doc for this:
[https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions)

Reply:
https://docs.microsoft.com/en-us/rest/api/cognitiveservices/accountmanagement/accounts

Reply:
Got that implemented and pushed. The docs were super helpful, thank you!